p2p/libp2p/netMessenger.go

```diff
@@ -115,6 +120,10 @@ type ArgsNetworkMessenger struct {
 
 // NewNetworkMessenger creates a libP2P messenger by opening a port on the current machine
 func NewNetworkMessenger(args ArgsNetworkMessenger) (*networkMessenger, error) {
+	return newNetworkMessenger(args, true, true)
+}
+
+func newNetworkMessenger(args ArgsNetworkMessenger, withMessageSigning bool, reusePort bool) (*networkMessenger, error) {
```

Comment:
can you replace the bool parameters with types? 
```
type messageSigningConfig bool
const (
    withMessageSigning messageSigningConfig = true
    withoutMessageSigning messageSigningConfig = true
)
```
or something similar

Reply:
done